.circleci/config.yml

```diff
@@ -18,19 +18,22 @@ jobs:
     <<: *docker_build
     steps:
       - checkout
-      - setup_remote_docker
       - run:
           name: Build Premkit
           command: |
             export BUILD_SHA=${CIRCLE_SHA1:0:7}
-            export PREMKIT_TAG=${CIRCLE_TAG}
+            export BUILD_VERSION=${CIRCLE_TAG//v}
```

Comment:
What's this syntax? What's getting replaced here?

Reply:
v1.2.3 = 1.2.3